app/alarm/ui/src/main/java/org/phoebus/applications/alarm/ui/annunciator/Annunciator.java

```diff
@@ -1,51 +1,20 @@
-/*******************************************************************************
- * Copyright (c) 2018 Oak Ridge National Laboratory.
- * All rights reserved. This program and the accompanying materials
- * are made available under the terms of the Eclipse Public License v1.0
- * which accompanies this distribution, and is available at
- * http://www.eclipse.org/legal/epl-v10.html
- *******************************************************************************/
 package org.phoebus.applications.alarm.ui.annunciator;
 
-import com.sun.speech.freetts.Voice;
-import com.sun.speech.freetts.VoiceManager;
-
-/**
- * Annunciator class. Uses freeTTS to annunciate passed messages.
- * @author Evan Smith
- */
-@SuppressWarnings("nls")
-public class Annunciator
+public interface Annunciator
 {
-    private final VoiceManager voiceManager;
-    private final Voice        voice;
-    private static final String voice_name = "kevin16";
-
-    /** Constructor */
-    public Annunciator()
-    {
-        // Define the voices directory.
-        System.setProperty("freetts.voices", "com.sun.speech.freetts.en.us.cmu_us_kal.KevinVoiceDirectory");
-        voiceManager = VoiceManager.getInstance();
-        voice = voiceManager.getVoice(voice_name);
-        voice.allocate();
-    }
+    /**
+     * Initialize the Annunciator resources.
+     */
+    default void initialize(){}
```

Comment:
In the new Annunciator SPI interface, I don't see how `initialize()` is ever called. If it's not needed, remove? Implementations can initialize in their constructor?

Reply:
> Implementations can initialize in their constructor?

Ya, that is what the freetts implementation does too... I just thought for the sake so  symmetry to have `initialize` to mirror the `shutdown`. I can remove this.